vllm/worker/vineyard_llm_cache.py

```diff
@@ -228,19 +221,17 @@ def prefetch_kv_caches(
         '''
         if block_size is None or kv_caches[0] is None:  # profile run
             return {}
-
         if seq_group_metadata_list is not None:
             prefill_requests = []
             for seq_group_meta in seq_group_metadata_list:
                 if seq_group_meta.is_prompt:
                     prefill_requests.append(seq_group_meta)
             num_prefill_requests = [len(prefill_requests)]
-            # torch.distributed.broadcast_object_list(num_prefill_requests, src=0,
-            #                                         group=get_tensor_model_parallel_group())
+            group = get_tp_group()
+            group.broadcast_object_list(num_prefill_requests, src=0)
```

Comment:
Might need to init `prefill_requests` after broadcast for other workers (rank!=0)